python-backend/tests/parties/party/resources/test_party_resource.py

```diff
@@ -93,20 +93,6 @@ def test_post_person_no_email(test_client, auth_headers):
     assert post_resp.status_code == 400
 
 
-def test_post_person_name_exists(test_client, auth_headers):
```

Comment:
Just wondering about removed tests, can your provide insight? where they duplicated? was the behavior no longer required?

Reply:
Some people have the same name. The unique name validation was removed (shh.. don't tell anyone). So, the tests had to go as well

Reply:
There is a ticket for that behavior (https://bcmines.atlassian.net/browse/MDS-1393) and larger issues that need to be considered, like how do we display a list of parties when we need more than just name to uniquely identify. I agree the constraint needs to go, but i'd like to see that problem tackled as a whole. Up to Aaron ultimately if he wants the constraint removed without the additional changes.

Reply:
I'm fine with reverting this change. I don't think this is a validation that should be in the codebase, though. It seems more problematic than anything. The choice to remove it was made during a conversation with @nayr974 and @mmumby this morning.

Reply:
I totally agree the validation rule shouldn't exist, but I don't support that validation being removed without a discussion with Aaron and explanation of the impact, as there is already a prioritized item in the backlog that contains this change.

Reply:
Reverted